src/System.Private.ServiceModel/src/System/IdentityModel/Tokens/GenericXmlSecurityKeyIdentifierClause.cs

```diff
@@ -0,0 +1,79 @@
+using System.IdentityModel.Tokens;
+using System.ServiceModel;
```

Comment:
Yes, it was to get DiagnosticUtility to resolve

---
In reply to: [78141546](https://github.com/dotnet/wcf/pull/1515#discussion_r78141546) [](ancestors = 78141546)